src/main/java/com/AdelaQA/classesandobjects/autoservicetask/Car.java

```diff
@@ -0,0 +1,22 @@
+package com.AdelaQA.classesandobjects.autoservicetask;
+
+public class Car {
+int kmWhenEnteredTheService;
```

Comment:
please apply: Ctrl + Alt + L to every java file, its shortcut for prettier code format